src/Microsoft.DotNet.Darc/src/DarcLib/Microsoft.DotNet.DarcLib.csproj

```diff
@@ -26,6 +26,7 @@
   <ItemGroup>
     <ProjectReference Include="..\..\..\Maestro\Client\src\Microsoft.DotNet.Maestro.Client.csproj" />
     <ProjectReference Include="..\..\..\Maestro\Maestro.Contracts\Maestro.Contracts.csproj" />
```

Comment:
Shouldn't also this be removed somewhere?

Reply:
removed